src/main/java/org/folio/search/service/metadata/SearchFieldProvider.java

```diff
@@ -20,9 +20,17 @@
   /**
    * Provides list of fields for given search type.
    *
-   * @param searchType search type as {@link String}
    * @param resource resource type as {@link String}
+   * @param searchType search type as {@link String}
    * @return list of fields.
    */
   List<String> getFields(String resource, String searchType);
+
+  /**
+   * Provides list of fields of source fields for resource.
+   *
+   * @param resource resource type as {@link String}
+   * @return list of fields.
+   */
```

Comment:
I don't think this comment adds more information, I'd prefer to remove it if that's fine.

Reply:
it's default javadoc structure - Header of javadoc, @param - for each parameter of method and return value.

Reply:
Sure, AFAIK on FOLIO it is not required to have comments if method name is clear enough.

Reply:
but it does not bring any issues in further, anyone can extend javadoc when implementation become more complicated or will require more detailed description

From perspective of checkstyle - javadoc is optional, but my IDE is set up to generate default stubs for any public method, so I just fill required fields in this template

Reply:
Usually folks (including me) forgot to update java docs :(. But that's fine, let's leave them as they are.